README.md

```diff
@@ -7,6 +7,20 @@ The interface is available at: [https://paritytech.github.io/bridge-ui](https://
 
 ## Building and running locally
 
+### Environment variables
+
+Make sure to setup an .env file to select the Substrate and Ethereum nodes you want to connect to, using the following variables:
```

Comment:
Is it required or optional?

Reply:
it's optional because we have default values set if the env is undefined. I'll mention that.